subcommand/injector/command.go

```diff
@@ -318,18 +320,21 @@ func getAdminAPIVersion(ctx context.Context, clientset *kubernetes.Clientset) (s
 
 func (c *Command) certWatcher(ctx context.Context, ch <-chan cert.Bundle, clientset *kubernetes.Clientset, leaderElector leader.Elector, adminAPIVersion string, log hclog.Logger) {
 	var bundle cert.Bundle
+	webhooksWatcher, err := watchWebhooks(ctx, clientset, adminAPIVersion)
+	if err != nil {
+		log.Warn("Error watching webhooks", "error", err)
+		webhooksWatcher = newTickerWatcher(1 * time.Second)
+	}
```

Comment:
This is falling back to the old method of patching once a second? (Just making sure I follow.)

Reply:
Yep! I figured if something was wrong, like we don't have permission to WATCH that endpoint or something, go back to the old behavior.